IR_Turret/IR_Turret.ino

```diff
@@ -1,8 +1,9 @@
 #include <Arduino.h>
 #include <Servo.h>
 
-#include "IR_Signals.h"
-#include "IR_Logic.h"
+#define DECODE_NEC
+#include "PinDefinitionsAndMore.h"
```

Comment:
This might not be completely necessary, but it is nice to have some extra functions to specify IR pins for different boards and hardware.